<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableView" name="DataView"/>
      </item>
      <item>
       <widget class="QGraphicsView" name="ShowView"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="loadDataBtn">
          <property name="text">
           <string>导入数据</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="featSelBtn">
          <property name="text">
           <string>特征筛选</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ModelTrainBtn">
          <property name="text">
           <string>模型建立</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ResultsBtn">
          <property name="text">
           <string>结果展示</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
